src/SUMMARY.md

```diff
@@ -83,4 +83,5 @@
       - [Global allocators](rust-2018/platform-and-target-support/global-allocators.md)
       - [MSVC toolchain support](rust-2018/platform-and-target-support/msvc-toolchain-support.md)
       - [MUSL support for fully static binaries](rust-2018/platform-and-target-support/musl-support-for-fully-static-binaries.md)
-      - [`cdylib` crates for C interoperability](rust-2018/platform-and-target-support/cdylib-crates-for-c-interoperability.md)
\ No newline at end of file
+      - [`cdylib` crates for C interoperability](rust-2018/platform-and-target-support/cdylib-crates-for-c-interoperability.md)
+    - [2018-Specific Changes](rust-2018/edition-changes.md)
```

Comment:
Might want to move this to be the first item, i.e. 3.1 (and modules become 3.2).